src/desktop/main.cc

```diff
@@ -353,6 +375,29 @@ MAIN {
         return;
       }
 
+      if (message.name == "send") {
+        if (message.index >= 0) {
+          auto window = windowFactory.getWindow(message.index);
+          if (window) {
+            window->eval(getEmitToRenderProcessJavaScript(
+              decodeURIComponent(message.get("event")),
+              value
+            ));
+          }
+        } else {
+          for (auto w : windowFactory.windows) {
+            if (w != nullptr) {
```

Comment:
`w` is a `Window*` so no need to call `getWindow()` below as `window == w` here

Reply:
> `w` is a `Window*` so no need to call `getWindow()` below as `window == w` here

it was `WindowWithMetadate*`